net/table_net_certificate.go

```diff
@@ -1,51 +1,79 @@
 package net
 
 import (
+	"bytes"
 	"context"
+	"crypto/ecdsa"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/tls"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/json"
 	"errors"
 	"fmt"
+	"io"
+	"io/ioutil"
 	"net"
+	"net/http"
 	"time"
 
-	"github.com/genkiroid/cert"
+	"golang.org/x/crypto/ocsp"
 
 	"github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
 	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
 	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
 )
 
+type OCSP struct {
+	StatusString           string     `json:"status"`
+	RevokedAt              *time.Time `json:"revoked_at,omitempty"`
+	RevocationReasonString string     `json:"revocation_reason,omitempty"`
+}
+
+//// TABLE DEFINITION
+
 func tableNetCertificate(ctx context.Context) *plugin.Table {
 	return &plugin.Table{
 		Name:        "net_certificate",
 		Description: "Certificate details for a domain.",
 		List: &plugin.ListConfig{
-			Hydrate:    tableNetCertificateList,
-			KeyColumns: plugin.SingleColumn("domain"),
+			Hydrate: tableNetCertificateList,
+			KeyColumns: plugin.KeyColumnSlice{
+				{Name: "domain", Require: plugin.Required, Operators: []string{"="}},
+			},
 		},
 		Columns: []*plugin.Column{
 			// Top columns
 			{Name: "domain", Type: proto.ColumnType_STRING, Description: "Domain name the certificate represents."},
 			{Name: "common_name", Type: proto.ColumnType_STRING, Description: "Common name for the certificate."},
 			{Name: "not_after", Type: proto.ColumnType_TIMESTAMP, Description: "Time when the certificate expires. Also see not_before."},
+			{Name: "is_revoked", Type: proto.ColumnType_BOOL, Hydrate: getRevocationInformation, Description: "Indicates whether the certificate was revoked, or not."},
+			{Name: "transparent", Type: proto.ColumnType_BOOL, Hydrate: getCertificateTransparencyLogs, Transform: transform.FromValue(), Description: "Indicates whether certificate is visible in certificate transparency logs."},
```

Comment:
The columns above and below this start with `is_`, but this one doesn't, even though it contains similar data types. How did you determine whether or not to use `is_` for these 3 columns?